packages/web-runtime/src/components/FeedbackLink.vue

```diff
@@ -9,7 +9,7 @@
       :aria-label="ariaLabel"
       aria-describedby="oc-feedback-link-description"
     >
-      <oc-icon name="feedback" />
+      <oc-icon name="feedback" variation="inverse" />
```

Comment:
Will not be `inverse` anymore as soon as the redesign is complete. Ok as intermediate solution.